docs/mkdocs.yml

```diff
@@ -0,0 +1,30 @@
+site_name:        GLM.jl
```

Comment:
This file isn't needed nor used with the standard setup where the native HTML backend is used. See for example in CategoricalArrays: https://github.com/JuliaData/CategoricalArrays.jl/blob/master/docs/make.jl